xesmf/tests/test_frontend.py

```diff
@@ -173,7 +174,7 @@ def test_to_netcdf(tmp_path):
     # Let the frontend write the weights to disk
     xfn = tmp_path / 'ESMF_weights.nc'
     method = 'bilinear'
-    regridder = xe.Regridder(ds_in, ds_out, method)
+    regridder = xe.Regridder(ds_in, ds_out, method, default_weights=False)
```

Comment:
I would also add the arg in the parameterize l 107 to check the new arg does not break tests

Reply:
also is the test passing for both default_weights = False AND True?

Reply:
`test_build_regridder` is passing for both settings, I will add the argument there.